how-to/customize-workspace/client/src/salesforce.ts

```diff
@@ -127,23 +126,15 @@ export interface SalesforceSettings {
   };
 }
 
-export async function openConnection(
-  settings?: SalesforceSettings
-): Promise<SalesforceConnection> {
-  if (!sfConn && settings) {
+export async function salesForceConnect(settings?: SalesforceSettings): Promise<void> {
+  if (settings?.orgUrl) {
```

Comment:
Shouldn't we still  check to see if an existing connection is already setup?

Reply:
Yes it should, I also need to double check that when there are errors it is cleaned up, so that it reconnects properly.